rules/sort-imports.ts

```diff
@@ -1,5 +1,10 @@
 import type { TSESTree } from '@typescript-eslint/types'
 
+import {
+  isExternalModuleNameRelative,
+  resolveModuleName,
+  sys,
+} from 'typescript'
```

Comment:
Wouldn't this break the use of the plugin in JavaScript projects?

Reply:
@azat-io True! I can make the imports dynamic so the tsconfig check isn't done in non-typescript projects. What do you think?

Reply:
I don't see any other way...

Reply:
It's actually a bit more annoying than I thought: dynamic imports are async, and there can be no async operation within the ESLint ecosystem.

Reply:
Using `createRequire`?

Reply:
@OlivierZal I wasn't using `createRequire`! I was going for the `await import` route, but this seems way better, thanks!